server/routerlicious/packages/services/src/factory.ts

```diff
@@ -0,0 +1,32 @@
+import { IDbFactory } from "@fluidframework/server-services-core";
+import { Provider } from "nconf";
+import registerDebug from "debug";
+
+const debug = registerDebug("fluid:backend");
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class DbFactoryFactory {
```

Comment:
There is already a very similar DB factory in tinylicious (server/tinylicious/src/services/dbFactory.ts), which creates the DB based on the setting for "db:inMemory". Could those be merged into one factory (and support for level DB added here)?